app/controllers/application_controller.rb

```diff
@@ -80,6 +80,7 @@ def authenticate_token
         ::Digest::SHA256.hexdigest(@session.token)
       )
       @current_user = User.find(@session.uuid)
```

Comment:
Could this and the next line be encapsulated in the User class, maybe a `User.from_session(session)` method? `@current_user.session = @session` pops up in a couple controllers.